packages/hub/routes/reward-proofs.ts

```diff
@@ -0,0 +1,60 @@
+import { inject } from '@cardstack/di';
+import Koa from 'koa';
+import autoBind from 'auto-bind';
+import Logger from '@cardstack/logger';
+import { isAddress } from 'web3-utils';
```

Comment:
I think you can get this util from the web3 package.

`import Web3 from 'web3';`

`Web3.utils.isAddress`

Reply:
Noted. Will follow suggestion